device-types/FS/IES3110-8TF-R

```diff
@@ -0,0 +1,122 @@
+---
+manufacturer: FS
+model: IES3110-8TF-R
+slug: ies3110-8tf-r
```

Comment:
All slugs should have the manufacturer appended to them, please append `fs-` in this case

Reply:
I saw your changes. I will make sure future PRs for different hardware have the updated changes.